__tests__/config/filters/isSource.test.ts

```diff
@@ -0,0 +1,21 @@
+import StyleDictionary from 'style-dictionary'
+import {isSource} from '../../../config/filters/isSource'
+
+describe('Filter: isSource', () => {
+  const items = [
+    {
+      value: 'red is source',
+      isSource: true
+    },
+    {
+      value: 'blue is not source',
+      isSource: false
+    },
+    {
+      value: 'yellow is not source'
+    }
+  ] as StyleDictionary.TransformedToken[]
+  it('filters out non-source items', () => {
+    expect(items.filter(isSource)).toStrictEqual([items[0]])
+  })
```

Comment:
A suggestion to improve tests like this.. 

You're asserting against a very specific position in the test fixture, but that value can change at any time and invalidate the test. It also doesn't capture unhappy paths or edge cases where multiple  truthy results are present in the fixture. 

```suggestion
  const result = items.filter(isSource)
  const expectedValue = items.filter(item => item.isSource)
  
  it('filters out non-source items', () => {
    expect(result).toStrictEqual(expectedValue)
  })
```

This should still achieve the same thing but now isn't dependent on the fixture data being correct, but also keeps `isSource` honest against regressions.

Ideally this would be done with property testing paradigms, but this should work in the interim.